playbooks/test_deploy_cukinia_tests.yaml

```diff
@@ -9,6 +9,12 @@
       copy:
         src: ../src/cukinia-tests/cukinia
         dest: /etc/
+    - name: Copy Cukinia's tests templates
+      template:
```

Comment:
apt.conf is not a template. I suggest using copy instead of template.

Reply:
Ok I was wrong, it is a template. Maybe we can rename the file to add the .j2 extension.